json_parser/idf_component.yml

```diff
@@ -1,5 +1,8 @@
-version: "1.0.2"
+version: "1.0.3"
```

Comment:
@igrr , do we need to change component version just for updating dependency?

Reply:
Yes, we do. A component will get published to the registry only when its version is changed. It's not possible to "overwrite" an already published version of the component.